src/extensions/modtools.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, bot):
         inumber = int(number)
         if inumber <= 100:
             # In order to delete the command message too, the number of messages to clear is incremented
+            # noinspection PyUnresolvedReferences

```

Comment:
Actually, don't bother with this one, about to push with another commit with some more type hinting for the util functions.